Source/DafnyCore/Dafny.atg

```diff
@@ -1864,7 +1883,7 @@ TPCharOption<ref TypeParameter.TypeParameterCharacteristics characteristics>
                       characteristics.AutoInit = Microsoft.Dafny.Type.AutoInitInfo.Nonempty;
                     }
                   } else {
-                    SemErr(t, "unexpected type parameter option - should be == or 0 or !new");
+                    SemErr(t, "unexpected type parameter option - should be == or 0 or 00 or !new");
```

Comment:
I think the `-` in this sentence is intended to work like an mdash. Alas, the sentence looks like it's asking the user to replace a `-` with an `==`, `0`, `00`, or `!new`. I suggest:
```suggestion
                    SemErr(t, "unexpected type parameter option; such an option should be == or 0 or 00 or !new");
```

Reply:
Revised differently to include the offending type parameter

Reply:
Looks good now